scripts/v8-to-v9.py

```
@@ -0,0 +1,178 @@
+#! /usr/bin/env python3
+
+import math
+import json
+import argparse
+from typing import Any
+
+def upgrade_genesis(data: Any, blank_path,dogfood_addr,chain_id: str) -> Any:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)